include/openenclave/bits/sgx/sgxproperties.h

```diff
@@ -114,7 +119,8 @@ typedef struct _oe_sgx_enclave_properties
     EXTENDED_PRODUCT_ID,                                                  \
     FAMILY_ID,                                                            \
     ALLOW_DEBUG,                                                          \
-    REQUIRE_KSS,                                                            \
+    REQUIRE_KSS,                                                          \
+    CAPTURE_PF_GP_EXCEPTIONS,                                             \
```

Comment:
Seems the comments for this macro need to be updated to add PFGP description.

Reply:
Thanks for the catch! Updated.